src/schema/SchemaSelectors.js

```diff
@@ -159,3 +160,30 @@ export const hasSchemaProperty = createSelector(
   [hasProperty],
   hasProperty => hasProperty
 );
+
+export const getActions = createSelector(
+  [actions],
+  actions => actions
+);
+
+export const getDetailViewActions = createSelector(
+  [actions],
+  actions => Object.keys(actions).reduce((result, action) => {
+    if (actions[action].path.startsWith('/:id/')) {
```

Comment:
it's should be `(/\/\:id\//).test(actions[action].path)` because we don't have guarantee of path convention.